chart/compass/charts/director/templates/external-certificate-rotation-job.yaml

```diff
@@ -113,16 +113,22 @@ spec:
                     -d "grant_type=client_credentials&token_format=bearer&client_id=$CERT_SVC_CLIENT_ID" \
                     | jq -r .access_token)
 
+                  if [[ -z "$TOKEN" ]]; then
```

Comment:
Do we want this check in the PR to the main branch?

Reply:
It is already in the main branch. I just decided to introduce it in the hotfix branch in case we need to debug something in the current release
https://github.com/kyma-incubator/compass/blob/b7b5279bf6a58933b2831dbf3063608526bd48ad/chart/compass/charts/director/templates/external-certificate-rotation-job.yaml#L116